.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,15 @@
+name: Autobump version
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  autobump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: bump
+        run: ./bump_version.sh && git push -f origin master || echo 'No action needed.'
```

Comment:
there seems to be a packaged Action for commits and pushes:
https://github.com/marketplace/actions/add-commit

maybe we could just use that? Simply define it as a step after bumping the version in the Cargo file?

Reply:
I have seen that, but I didn't see a way of doing amends there – and creating an additional commit after every merge seems like a mess.